src/main/java/com/example/demo/model/Book.java

```diff
@@ -0,0 +1,30 @@
+package com.example.demo.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Column;
+import lombok.Data;
+import java.math.BigDecimal;
+
+@Entity
+@Data
```

Comment:
don't use `@Data` for entities
Add `@Table` with name
```suggestion
@Getter
@Setter
```